app/Console/Commands/CleanAcarsMessages.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\Acars\AcarsMessage;
+use Carbon\Carbon;
+use Illuminate\Console\Command;
+
+class CleanAcarsMessages extends Command
+{
+    protected $signature = 'acars:clean-history';
+
+    protected $description = 'Delete any acars message history that is older than a specified age';
+
+    public function handle(): int
+    {
+        AcarsMessage::where(
+            'created_at',
+            '<',
```

Comment:
`<=` ?

Reply:
We've consistently used < on other similar jobs, so might be good to stay consistent?